### DISCUSSION TITLE
How to cache standard images?

### DISCUSSION THREAD

**Discussion by acidos** [community/community#25373] [Actions]
<p>In my CI/CD workflow I need to use database container, which is standard <code>postgres:alpine</code> image. I don’t need to build new image based on it, but just run it with specific environment options.</p>
<p>Downloading every time the image from docker hub is time consuming.<br>
How can I cache it somewhere, so instead of downloading, it will be available locally from one workflow run to another?</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/acidos">@acidos</a>,</p>
<p>According to the official doc of <a href="https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainerimage" rel="nofollow noopener">container.images</a>:<br>
<em>The value can be the Docker Hub image name or a public docker registry name.</em></p>
<p>You cannot store the container image locally. You have to download it from docker hub…etc.</p>
<p>Thanks.</p>
